src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const str = sourceString.split(';');
+  const result = {};
+
+  str.map((element) => {
+    if (element.length >= 2) {
+      const splitElement = element.split(':').map((item) => item.trim());
+
+      result[splitElement[0]] = splitElement[1];
+    }
+  });
```

Comment:
it is not a proper usage for map method. Map should return some array. Use either forEach of reduce method here